client/src/views/Simulation/components/ParametersPanel.vue

```diff
@@ -103,6 +103,7 @@
   export default class ParametersPanel extends Vue {
     @Prop({ default: false }) expanded: boolean;
     @Prop({ default: null }) modelId: number;
+    @Prop({ default: '' }) highlighted: string;
```

Comment:
Why not `null` in this case? Just curious.

Reply:
Not sure what's the convention we follow for default values actually. I'll change it to `null`